docs/modules/ROOT/pages/how-tos/cloudscale/add_node.adoc

```diff
@@ -58,12 +59,12 @@ include::partial$install/approve-node-csrs.adoc[]
 +
 [source,bash]
 ----
-kubectl get node
+kubectl --as=cluster-admin get node
```

Comment:
I think technically we don't need cluster-admin anymore to get nodes. But doesn't hurt either and might not be true on all clusters, so :shrug: